volatility/framework/plugins/windows/dumpfiles.py

```diff
@@ -0,0 +1,268 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
Should probably update this 2020, but not a show stopper...  5:P

Reply:
I can update the copywrite to 2020 on the files I updated. That's simple enough.